sdk/python/feast/feature_server.py

```diff
@@ -88,10 +90,15 @@ def get_online_features(body=Depends(get_body)):
             body = json.loads(body)
             # Initialize parameters for FeatureStore.get_online_features(...) call
             if "feature_service" in body:
-                features = store.get_feature_service(
+                feature_service = store.get_feature_service(
                     body["feature_service"], allow_cache=True
                 )
+                assert_permissions(
```

Comment:
Isn't this supposed to be called before line 106 ?

Reply:
The reason I have putted here inside if block because I am setting `feature_service` in resources which is feast object.  if move down then if it come into else block then it will set `features` which is not FeastObject and I can't set. 

The actual code before this changes was like this 
```              if "feature_service" in body:
                features = store.get_feature_service(
                    body["feature_service"], allow_cache=True
                )
            else:
                features = body["features"]
```

between I have raise the question in TODO else block what if it's feature lists how can set the security for that object.